src/langs/en/auth.ts

```diff
@@ -0,0 +1,14 @@
+import { AUTH } from '../../constants/auth';
```

Comment:
Isn't this file redundant w.r.t src/constants/auth.ts since it is default-based? See comment above. In general, when you use default-based translations, the original translation language does not need to have a specific translation file